modules/installation/proc-rhdh-deploy-eks-operator.adoc

```diff
@@ -4,7 +4,7 @@
 [id='proc-rhdh-deploy-eks-operator_{context}']
 = Installing {product-short} on {eks-short} with the Operator
 
-You can install the {product} Operator with or without the Operator Lifecycle Manager (OLM) framework.
+The {product} Operator installation requires the Operator Lifecycle Manager (OLM) framework.
 
 .Additonal resources
 * For information about the OLM, see link:https://olm.operatorframework.io/docs/[Operator Lifecycle Manager(OLM)] documentation.
```

Comment:
Leftovers. This information is already present line 22.

```suggestion
```

Reply:
Fixed as per https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/729#discussion_r1867617883